test/spec/Feature/Query/QueryLimitedSpec.hs

```diff
@@ -93,13 +93,16 @@ spec =
               { "id": 8, "name": "HaikuOS" } ]|]
             { matchStatus  = 201 }
 
-      it "doesn't affect updates(2 rows would be modified if it did)" $
+      it "doesn't affect updates" $
```

Comment:
The comment in the parens, it still applies right? I think that was related to https://github.com/PostgREST/postgrest/issues/2155

If true then that should be brought back.

Reply:
True, I was misunderstanding the test. Fixed.